test/unit/captureTasksController.spec.js

```diff
@@ -0,0 +1,54 @@
+describe('CaptureTasksController', function(){
+  beforeEach(module('Capture'));
+
+  var ctrl;
+
+  beforeEach(inject(function($controller){
+    ctrl = $controller('CaptureTasksController');
+  }));
+
+  it('initialises with an empty task load', function(){
+    expect(ctrl.loadTask).toBeUndefined();
+  });
+
+  it('can add a task to the start of the list', function(){
+    ctrl.addTask('A first Task');
+    ctrl.addTask('A second Task');
+    expect(ctrl.tasks[0].desc).toEqual('A second Task')
```

Comment:
Split out into two tests, maybe?

`it('can add a task')`
and
`it('adds tasks to the beginning of the list`)`